tests/testflows/extended_precision_data_types/configs/clickhouse/config.d/ssl.xml

```diff
@@ -0,0 +1,18 @@
+<yandex>
```

Comment:
Garbage?

Reply:
Yes. Will remove the unnecessary SSL files.